languages/python/exercises/concept/none/.docs/after.md

```diff
@@ -0,0 +1,28 @@
+All variables that are assigned `None` point to the same object. New instances of `None` are not created, that is `None` by nature is of type Singleton.
+
+Example, when you assign 2 variables with `None`, both point to the same object.
+
+```python
+a = None
+b = None
+a == b
+#=> True
+```
+
+The `None` Object is a singleton of class `NoneType`. So when you find the type of a variable which is assigned with `None` it will always be of type `NoneType`.
+
+```python
+a = None
+type(a)
+#=> <class 'NoneType'>
+```
+
+When you toggle a variable between a value and `None`, what basically happens is a reset of the variable.
+
+```python
+
+a = [] #=> Variable 'a' is pointing to a list
+
+a = None #=> Variable 'a' is reset to an empty state. the value of a is now absent.
+
+```
```

Comment:
We might also want to add some discussion here around what you need to do when you can't use `None` as your only placeholder.  Check out this [None](https://realpython.com/null-in-python/) article from Real Python.

Reply:
I may need to come back to this one again